runtime/src/snapshot_utils.rs

```diff
@@ -277,6 +277,91 @@ pub enum VerifySlotDeltasError {
     BadSlotHistory,
 }
 
+/// Delete the files and subdirectories in a directory.
+/// This is useful if the process does not have permission
+/// to delete the top level directory it might be able to
+/// delete the contents of that directory.
+fn delete_contents_of_path(path: impl AsRef<Path> + Copy) {
```

Comment:
nit: Why did we reverse the order of these functions? All the other pub/helper pairs in this file are in the order: pub first, helper(s) immediately below.

Reply:
I copied the whole block from validator.rs to this file.  I guess there is no such order convention in that file.  I can reverse it here.  But I think this kind of convention does not matter, and is hard to maintain.  When a function is changed from private to public, is there need to generate a big diff just to maintain this order?

Reply:
Yeah, was just wondering why the order did get reversed. I tried checking the new vs old blocks by just ctrl-Fing the entire moved code block, which failed due to the fn re-ordering.